tensorflow/tools/ci_build/windows/cpu/pip/build_tf_windows.sh

```diff
@@ -32,28 +32,11 @@ set -e
 # This script is under <repo_root>/tensorflow/tools/ci_build/windows/cpu/pip/
 # Change into repository root.
 script_dir=$(dirname $0)
-cd ${script_dir%%tensorflow/tools/ci_build/windows/cpu/pip}
+cd ${script_dir%%tensorflow/tools/ci_build/windows/cpu/pip}.
```

Comment:
no, I mean there is now an extra dot (.) at the end of the line.
That is probably not needed.